frontend/src/components/core/MainMenu/MainMenu.tsx

```diff
@@ -246,6 +246,7 @@ function MainMenu(props: Props): ReactElement {
   const showClearCache = !shouldShowS4AMenu
   const preferredMenuOrder: any[] = [
     coreMenuOptions.rerun,
+    shouldShowS4AMenu && coreMenuOptions.settings,
```

Comment:
I don't think it really matters since cache is disabled for S4A but according to the original spec, settings goes after cache.

Reply:
Oh derp. Since it doesn't make a visual difference and I don't want to wait for the tests again, I'm just merging as-is :P